### PR TITLE
docs: fixes to the Image upload documentation

### PR DESCRIPTION
**Pull request by teolemon**
### What
docs: fixes to the Image upload documentation

### CODE REVIEW

File: docs/api/tutorial-uploading-photo-to-a-product.md [hunk 1]
```diff
@@ -6,9 +6,15 @@ Be sure to also read the [introduction to the API](./index.md)
 
 ## Points to consider before uploading photos
 
+### Why photos are important on Open Food Facts
+* Photos are source and proof of data.
+* They build trust in your edit.
+* They are the fastest and laziest way for your users to contribute
```
Comment (hangy) [inline docs/api/tutorial-uploading-photo-to-a-product.md:12]: I don't know if "lazy" is the best way to word it. I think it usually has a pretty negative connotation? Maybe a native speaker could have a look?

File: docs/api/tutorial-uploading-photo-to-a-product.md [hunk 2]
```diff
@@ -6,9 +6,15 @@ Be sure to also read the [introduction to the API](./index.md)
...
+
 ### Image license
 
 Product images must be under the [Creative Commons Attribution ShareAlike licence](https://creativecommons.org/licenses/by-sa/3.0/deed.en).
+If you don't own the image, don't upload it. Do not upload scrapped content from the internet. Only upload content taken by final users. If you have content from producers, please get in touch with producers@openfoodfacts.org before any upload.
```
Comment (hangy) [inline docs/api/tutorial-uploading-photo-to-a-product.md:17]:
```suggestion
If you don't own the image, don't upload it. Do not upload scraped content from the internet. Only upload content taken by final users. If you have content from producers, please get in touch with producers@openfoodfacts.org before any upload.
```

What do you mean by "final" users?

File: docs/api/tutorial-uploading-photo-to-a-product.md (hunk 1 above)
Comment (teolemon) [reply docs/api/tutorial-uploading-photo-to-a-product.md:12]: easiest might be more consensual ?

**Review by stephanegigandet — APPROVED** [review]
See hangy's comment, "easiest" is better I think.

File: docs/api/tutorial-uploading-photo-to-a-product.md (hunk 1 above)
Comment (teolemon) [inline docs/api/tutorial-uploading-photo-to-a-product.md:12]:
```suggestion
* They are the fastest and easiest way for your users to contribute
```

**Pull request merged**